tests/core/test_impersonation_tests.py

```diff
@@ -114,17 +114,19 @@ def create_user():
 
 @pytest.mark.quarantined
 class TestImpersonation(unittest.TestCase):
-    def setUp(self):
-        check_original_docker_image()
-        grant_permissions()
-        add_default_pool_if_not_exists()
-        self.dagbag = models.DagBag(
+    @classmethod
+    def setUpClass(cls):
+        cls.dagbag = models.DagBag(
```

Comment:
Not a required change, but sharing the DAG bag across tests provides a slight bit of performance gain. Most test case classes actually uses this pattern, not sure why it’s not applied here.